_posts/platform/deployment/2000-01-01-cache.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Deployment Cache
+nav: Cache
+modified_at: 2018-08-10 00:00:00
+tags: cache, deployment, reset
+---
+
+To make your builds faster, Scalingo uses a cache named the Deployment cache.
+Depending on your technology, many different things might go into this cache.
+
+For example, in NodeJS, by default, we cache the `.node_modules` directory so they will be available on your next deployment and we wont have to download your entire dependency tree for each deployment.
+
+{% note %}
+Protip: If you are a buildpack developer, you can store things in this cache by moving files into the `CACHE_DIR` folder passed as the second argument of the `bin/compile` script.
+{% endnote %}
+
+## Clearing your deployment cache
+If you need to clean your deployment cache, you can use our CLI tool:
```

Comment:
I would add a sentence explaining in which case clearing the cache can be useful